crates/egui_demo_lib/src/demo/widget_gallery.rs

```diff
@@ -300,15 +300,12 @@ mod tests {
             date: Some(chrono::NaiveDate::from_ymd_opt(2024, 1, 1).unwrap()),
             ..Default::default()
         };
-        let app = |ctx: &Context| {
-            CentralPanel::default().show(ctx, |ui| {
-                demo.ui(ui);
-            });
-        };
-        let harness = Harness::builder()
-            .with_size(Vec2::new(380.0, 550.0))
+        let mut harness = Harness::builder()
             .with_dpi(2.0)
```

Comment:
I should have caught this in an earlier PR, but this should be called `with_pixels_per_point`. `dpi` means `Dots per inch`, and is an absolute measurement of pixel density